dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/master/dispatch/executor/NettyExecutorManager.java

```diff
@@ -133,26 +136,13 @@ public void executeDirectly(ExecutionContext context) throws ExecuteException {
      * @throws ExecuteException if error throws ExecuteException
      */
     private void doExecute(final Host host, final Command command) throws ExecuteException {
-        /**
-         * retry count，default retry 3
-         */
-        int retryCount = 3;
-        boolean success = false;
-        do {
-            try {
+        try {
+            RetryerUtils.retryCall(() -> {
                 nettyRemotingClient.send(host, command);
-                success = true;
-            } catch (Exception ex) {
-                logger.error(String.format("send command : %s to %s error", command, host), ex);
-                retryCount--;
-                try {
-                    Thread.sleep(100);
```

Comment:
this sleep 100, retryer sleep 1000, Does 1000 affect second-level tasks?

Reply:
I think 100 or 1000 makes no sense, and doesn't matter. We can't say which interval is appropriate.

Reply:
If so, it is recommended to be consistent before and after reconstruction.
The original author may have his own consideration.

Reply:
@Technoboy- @qiaozhanwei  Could you give your consideration about this ? Is 1000 milli oK?

Reply:
> this sleep 100, retryer sleep 1000, Does 1000 affect second-level tasks?

1000 also will be OK